docs/using/selecting.md

```diff
@@ -16,27 +16,47 @@ This section provides details about the first.
 The Jupyter team maintains a set of Docker image definitions in the <https://github.com/jupyter/docker-stacks> GitHub repository.
 The following sections describe these images, including their contents, relationships, and versioning strategy.
 
+### jupyter/base-jupyter
+
+[Source on GitHub](https://github.com/jupyter/docker-stacks/tree/main/base-jupyter) |
+[Dockerfile commit history](https://github.com/jupyter/docker-stacks/commits/main/base-jupyter/Dockerfile) |
+[Docker Hub image tags](https://hub.docker.com/r/jupyter/base-jupyter/tags/)
+
+`jupyter/base-jupyter` is a small image supporting a majority of [options common across all core stacks](common.md).
+It is the basis for all other stacks on which Jupyter-related applications can be built (e.g., kernel-based containers,
+[nbclient](https://github.com/jupyter/nbclient) applications, etc.). As such, it does not
+contain application-level software like Jupyter Notebook server, Jupyter Lab or Jupyter Hub.
+
+It contains:
+
+- [Miniforge](https://github.com/conda-forge/miniforge) Python 3.x in `/opt/conda` with two package managers
```

Comment:
It doesn't contain Miniforge anymore, so this section is outdated. Let's fix it in this PR?